flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1062,6 +1067,9 @@ static void update_stabilization_desired(ManualControlCommandData * manual_contr
 		case FLIGHTSTATUS_FLIGHTMODE_ACRO:
 			stab_modes = RATE_SETTINGS;
 			break;
+		case FLIGHTSTATUS_FLIGHTMODE_ACRODYNE:
```

Comment:
Yah, that's pretty much universal for current code :(  Worse, since there's no default, the behavior is undefined.